docs/examples/1.4.x/console-cli/examples/health/get-queue-messaging.md

```diff
@@ -1 +1,2 @@
-appwrite health getQueueMessaging
```

Comment:
We should change the CLI naming conventions to use dash-case instead of camelCase.